src/frogger.cpp

```diff
@@ -58,20 +99,46 @@ Rcpp::List run_base_model(const Rcpp::List demp, SEXP sim_years) {
   Rcpp::NumericVector r_total_population(age_groups_pop * num_genders);
   Rcpp::NumericVector r_births(1);
   Rcpp::NumericVector r_natural_deaths(age_groups_pop * num_genders);
+  Rcpp::NumericVector r_hiv_population(age_groups_pop * num_genders);
+  Rcpp::NumericVector r_hiv_natural_deaths(age_groups_pop * num_genders);
+  Rcpp::NumericVector r_hiv_strat_adult(disease_stages * age_groups_hiv *
```

Comment:
Bit confused about how this line is not covered but the others are.. any ideas? Is the coverage for C++ just imperfect?

Reply:
Chatted about this with Rich, looks like codecov with C++ doesn't always produce most accurate reports. Don't need to worry too much about this missed line